src/js/components/Form/Form.js

```diff
@@ -233,6 +234,14 @@ const Form = forwardRef(
                 // which probably came from a submit
                 setErrors(prevErrors => ({ ...prevErrors, ...nextErrors }));
                 setInfos(prevInfos => ({ ...prevInfos, ...nextInfos }));
+
+                // give user access to errors that have occurred on validation
+                if (onValidate) {
+                  const validation = {};
+                  validation.error = { ...errors, ...nextErrors };
```

Comment:
It feels slightly odd that we are using `errors` here but `prevErrors` above on line 235. I wonder if we should merge `errors` and `infos` into the same state. For example:
```
const [validationResults, setValidationResults] = useState(defaultValidationResults);
...
setValidationResults(prevValidationResults => {
  const nextValidationResults = { errors: { ...prevValidationResults.errors, ...nextErrors }, infos: ... };
  if (onValidate) onValidate(nextValidationResults);
  return nextValidationResults;
});
```

Reply:
This is a nice idea. Let me try that